.travis.yml

```diff
@@ -1,14 +1,52 @@
 sudo: required
 
-language: java
+language: scala
 
 services:
   - docker
 
+# An alternative approach is to build a full matrix.
+#scala:
+#  - "2.10"
+#  - 2.11
+#
+#env:
+#  - KAFKA_VERSION=0.8.2.2
+#  - KAFKA_VERSION=0.9.0.1
+#  - KAFKA_VERSION=0.10.2.1
+#  - KAFKA_VERSION=0.11.0.2
+#  - KAFKA_VERSION=1.0.1
+#  - KAFKA_VERSION=1.1.0
+
+# This version will be also tagged as 'latest'
+env:
+  global:
+    - LATEST="2.11-1.1.0"
```

Comment:
This could also be parsed from the main Dockerfile to keep this info in one place. However, I thought this change was big enough and could see pros/cons to both approaches.